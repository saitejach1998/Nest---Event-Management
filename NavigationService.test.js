import React from 'react';
import navigate from './NavigationService.js';
import renderer from 'react-test-renderer';

describe('navigatePostEvents', function() {
    it('renders a Post event component ', function() {
      const tree = renderer.create(
        <navigate />
        ).toJSON();
    expect(tree).toMatchSnapshot();
    });
  });