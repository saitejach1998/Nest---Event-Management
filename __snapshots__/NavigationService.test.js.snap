// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`navigatePostEvents renders a Post event component  1`] = `<navigate />`;
